client/components/count/index.js

```diff
@@ -9,6 +9,11 @@ import PropTypes from 'prop-types';
  */
 import './style.scss';
 
+/**
+ * Display a number with a styled border.
+ *
+ * @return { object } -
```

Comment:
You'll see I had to add `@return` & `@param` definitions for all functional components. This is needed for eslint's `valid-jsdoc` rule, but jsdoc doesn't really apply here, since we describe the props in more detail below. We can leave it as a strange "eslint requirement", or convert all the functional components to classes, or remove the valid-jsdoc rule… thoughts?